cognitedata/reveal
# fix: Deleting models while sectors are loading no longer causes error flood

#### Type of change

![Bug](https://img.shields.io/badge/Type-Bug-red) 

#### Jira ticket :blue_book:

https://cognitedata.atlassian.net/browse/REV-415

## Description :pencil:
Since aborting the downloads early enough when a model deleted is not a trivial thing to do (see task for details), it has been deemed ok to log this error as an info.

## How has this been tested? :mag:

This has been tested by starting the Reveal Example with a large model and then, while the model is loading, used the GUI to remove last added model.

I have also tested this by adding more models by code using `modelUi.addModel` and repeated the process above. 

## Test instructions :information_source:

1. Run Reveal Examples with a large model.
2. While the model is still loading use the GUI and click the "Remove last model" option under Models:

![image](https://user-images.githubusercontent.com/89901292/218051370-1d219249-9d9e-4ad6-98cb-00414232c569.png)

Result: There should be no error spam.

## Checklist :ballot_box_with_check:
